SzczepionkaAPI/src/main/java/com/szczepionka/controller/AppointmentController.java

```diff
@@ -1,12 +1,11 @@
 package com.szczepionka.controller;
 
 import com.szczepionka.entity.Appointment;
+import com.szczepionka.model.AppointmentDetailsDTO;
 import com.szczepionka.model.PatientDTO;
 import com.szczepionka.service.AppointmentService;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Imports with * are not that good. Please change it back. 
![image](https://user-images.githubusercontent.com/18305244/113256421-755c5800-92c9-11eb-9f8c-2fb0cbfdae60.png)

Reply:
fixed